pkg/minikube/config/config.go

```diff
@@ -124,8 +124,11 @@ func encode(w io.Writer, m MinikubeConfig) error {
 }
 
 // Load loads the kubernetes and machine config for the current machine
-func Load() (*MachineConfig, error) {
+func Load(profile ...string) (*MachineConfig, error) {
```

Comment:
For consistency sake: let's just make the profile name a requirement.

Reply:
this will make the PR a bit bigger, but thats reasoanble aI will addd it